library/postfixadmin

```diff
@@ -2,17 +2,17 @@
 Maintainers: David Goodwin <david@codepoets.co.uk> (@DavidGoodwin)
 GitRepo: https://github.com/postfixadmin/docker.git
 
-Tags: 3.2.4-apache, 3.2-apache, 3-apache, apache, 3.2.4, 3.2, 3, latest
+Tags: 3.3.3-apache, 3.3-apache, 3-apache, 3.3.3, 3.3, 3
```

Comment:
Some tags like `latest` and `fpm-alpine` are being removed. Is that on purpose?

Reply:
Nope, thanks for the hint!